### PR TITLE
Fixup GeneralizedPlaneStrainAction

### PR DESCRIPTION
**Pull request by dschwen**
In preparation for TensorMechanicsAction work I need this action to be smart enough to handle temperature as an AuxVariable. I've also added comments and trimmed down the parameter passing using `applyParameters()`

Refs #7987

### CODE REVIEW

**Review by bwspenc — APPROVED** [review]

**Comment by bwspenc** [comment]
The code looks fine, although it apparently still has some issues that still need to be worked out since it's breaking some tests.

**Comment by dschwen** [comment]
I assume `combined:generalized_plane_strain_tm_contact.generalized_plane_strain_tm_contact` is an unrelated failure that was mentioned earlier in the standup, @bwspenc @sapitts ?

**Comment by bwspenc** [comment]
@dschwen The test failing here is not due to that issue we were talking about.  This is the error from exodiff:
exodiff: WARNING .. specified element variable "strain_zz" is not in the first file.
It sounds like you are adding strain_zz, but it's not in the gold file.

**Comment by dschwen** [comment]
Ah, got it thanks. This is probably due to the `aux_strain_zz` -> `strain_zz` renaming

File: framework/src/functions/MooseParsedFunction.C [hunk 1]
```diff
@@ -5,7 +5,7 @@
 /*           (c) 2010 Battelle Energy Alliance, LLC             */
 /*                   ALL RIGHTS RESERVED                        */
 /*                                                              */
-/*          Prepared by Battelle Energy Alliance, LLC           */
+/*          Prepared by Battelle Energy Alliadnce, LLC           */
```
Comment (jwpeterson) [inline framework/src/functions/MooseParsedFunction.C]: wut

Comment (permcody) [inline framework/src/functions/MooseParsedFunction.C]: 😆

**Review by permcody — CHANGES_REQUESTED** [review]
Let's fix this comment before we merge.

File: framework/src/functions/MooseParsedFunction.C (hunk 1 above)
Comment (dschwen) [inline framework/src/functions/MooseParsedFunction.C]: Ugh, I was testing the spell check after you asked me about spellchecking in code comments :-/

**Comment by permcody** [comment]
RunException interleaved output error - ignoring.

**Review by permcody — APPROVED** [review]

**Pull request merged**